### PR TITLE
Add scout-push

### PR DESCRIPTION
**Pull request by chrispatrick**

### CODE REVIEW

File: docker/ops.go [hunk 1]
```diff
@@ -186,39 +126,19 @@ func ProcessMessage(message *babashka.Message) (any, error) {
 				return nil, err
 			}
 			return shellquote.Split(args[0])
-		case "docker.tools/generate-sbom":
```
Comment (kipz) [inline docker/ops.go:189]: Should be removing the old generate-sbom? This could be useful IMHO.

File: docker/ops.go [hunk 2]
```diff
@@ -54,55 +56,11 @@ func parse_uri(s string) (Reference, error) {
 	return Reference{Path: path, Domain: domain, Tag: tag, Digest: digest}, err
 }
 
-func generate_sbom(message *babashka.Message, image string, username string, password string) error {
```
Comment (kipz) [inline docker/ops.go:57]: Why are we removing things?

**Review by kipz — COMMENTED** [review]
Some questions.

File: docker/ops.go (hunk 1 above)
Comment (chrispatrick) [reply docker/ops.go:189]: We either need to remove it or do a load of extra work. `generate-sbom` is using the old `index-cli-plugin` which would need updating in order to be able to import alongside `scout-cli-plugin.

File: docker/ops.go (hunk 2 above)
Comment (chrispatrick) [reply docker/ops.go:57]: It is the quickest solution. See other comment.

File: docker/ops.go (hunk 1 above)
Comment (slimslenderslacks) [reply docker/ops.go:189]: I've been using generate-sbom to transact datoms against a local store but I understand if this is no longer compatible or supported.

File: docker/ops.go (hunk 2 above)
Comment (chrispatrick) [reply docker/ops.go:57]: Or rather we can't use it without dealing with dependencies.

**Pull request closed without merging**